spec/models/like_spec.rb

```diff
@@ -0,0 +1,18 @@
+RSpec.describe Like, type: :model do
+  subject { Like.new(author_id: 1, post_id: 1) }
+
+  it 'increments likes_counter of post' do
+    # Create a user
+    user = User.create(name: 'Akite', posts_counter: 0)
+    # Create a post
+    post = user.posts.build(title: 'Post 1', comments_counter: 0, likes_counter: 0)
+    post.author_id = user.id
+    # Create a like
+    post.save!
+    like = post.likes.build(author_id: user.id)
+    like.post_id = post.id
+    # Increment likes_counter
+    like.save!
+    expect(Post.find(1).likes_counter).to eq(1)
+  end
```

Comment:
- [ ] @quavo19 Great work on your tests, however, this test is failing when I run tests as shown below, I would recommend  you check how you are creating the user post and like
![image](https://user-images.githubusercontent.com/61408860/236538882-a0117bb4-a0a8-4fc4-90ec-24ff58c85eff.png)